CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## Pending changes
 
-–
+- [#245](https://github.com/bumble-tech/appyx/pull/245) – **Fixed**: Card and spotlight-advanced navmodel samples now store their state.
```

Comment:
Let's not add changes which are not gonna be published to the changelist.

Reply:
We've previously done this for samples. Should we remove these in this PR?

Reply:
yes, and I think it was a mistake, and we removed them from the latest release https://github.com/bumble-tech/appyx/commit/e4cb7e66ff27c3cc43d5a74d176cf63db2517140

Reply:
@KovalevAndrey thanks! done :)